components/lcd/esp_lcd_lt8912b/idf_component.yml

```diff
@@ -0,0 +1,5 @@
+version: "0.1.0"
+description: ESP LCD LT8912B (MIPI DSI - HDMI)
```

Comment:
Could you please update https://github.com/espressif/esp-bsp/blob/master/LCD.md too?

Reply:
Updated.